pkg/models/stats.go

```diff
@@ -7,6 +7,7 @@ type SystemStats struct {
 	ActiveUsers               int64
 	DailyActiveUsers          int64
 	MonthlyActiveUsers        int64
+	DuplicateUserEntries      int64 `xorm:"duplicate_user_entries"`
```

Comment:
```suggestion
```

Reply:
Removed the line, I don't think you need it?

Reply:
oh right! 🤣